src/Employer/Employer.Web/Views/Error/AccessDenied.cshtml

```diff
@@ -12,7 +12,9 @@
                 <div class="inner">
                     <p>Your account does not have sufficient privileges</p>
                     <p>If you are experiencing difficulty accessing the area of the site you need, first contact an/the account owner to ensure you have the correct role assigned to your account.</p>
-                    <p><a href="/" class="govuk-button">Go back to the service homepage</a></p>
+                    <p>
+                        <a href="/" class="govuk-button">Go back to the service homepage</a>
+                    </p>
                 </div>
```

Comment:
Did you make change in this file? other than moving to a new line?

Reply:
This is to match the changes in the same files on the other sites